mlflow/utils/request_utils.py

```diff
@@ -0,0 +1,151 @@
+import os
```

Comment:
Can we add a test to verify that this module can be imported without mlflow being imported?

Reply:
Just added :)